examples/flutter_gallery/lib/demo/playground/configuration/material_helpers.dart

```diff
@@ -0,0 +1,297 @@
+  IconChoice(icon: Icons.accessibility, code: 'Icons.accessibility'),
+  IconChoice(icon: Icons.call, code: 'Icons.call'),
+  IconChoice(icon: Icons.camera, code: 'Icons.camera'),
+];
+
+final List<String> borderOptions =
+    borderChoices.map((BorderChoice b) => b.type).toList();
+
+final List<Color> colorOptions =
+    colorChoices.map((ColorChoice c) => c.color).toList();
+
+final List<IconData> iconOptions =
+    iconChoices.map((IconChoice i) => i.icon).toList();
+
+String codeSnippetForColor(Color color) {
+  return colorChoices
+      .where((ColorChoice c) => c.color == color)
+      .toList()[0]
+      .code;
+}
+
+String codeSnippetForBorder(String type) {
+  return borderChoices
+      .where((BorderChoice b) => b.type == type)
+      .toList()[0]
+      .code;
+}
+
+String codeSnippetForIcon(IconData icon) {
+  return iconChoices.where((IconChoice b) => b.icon == icon).toList()[0].code;
+}
+
+Widget sliderPicker({
+  @required String label,
+  double value = 0.0,
+  double minValue = 0.0,
+  double maxValue = 1.0,
+  int divisions,
+  ValueChanged<double> onValueChanged,
+}) {
+  return PropertyColumn(
+    label: label,
+    widget: Padding(
+      padding: const EdgeInsets.only(bottom: 5.0),
+      child: Slider(
+        value: value,
+        min: minValue,
+        max: maxValue,
+        divisions: divisions,
+        onChanged: onValueChanged,
+      ),
+    ),
+  );
+}
+
+ShapeBorder borderShapeFromString(String type, [bool side = true]) {
+  ShapeBorder shape;
+
+  BorderSide borderSide = const BorderSide(
+    color: Colors.grey,
+    width: 2.0,
+  );
+
+  borderSide = side ? borderSide : BorderSide.none;
+
+  switch (type) {
+    case 'square':
+      shape = RoundedRectangleBorder(
+          borderRadius: BorderRadius.zero, side: borderSide);
+      break;
+    case 'rounded':
+      shape = RoundedRectangleBorder(
+          borderRadius: BorderRadius.circular(10.0), side: borderSide);
+      break;
+    case 'beveled':
+      shape = BeveledRectangleBorder(
+          borderRadius: BorderRadius.circular(10.0),
+          side: borderSide.copyWith(width: 1.0));
+      break;
+    case 'stadium':
+      shape = StadiumBorder(side: borderSide);
+      break;
+  }
+  return shape;
+}
+
+Widget shapeButton({
+  @required String shape,
+  Size size = const Size(_pickerRowHeight, _pickerRowHeight),
+  bool isSelected = false,
+  VoidCallback onTapped,
+}) {
+  return ButtonTheme(
+    minWidth: size.width,
+    height: size.height,
+    child: RaisedButton(
+      shape: borderShapeFromString(shape),
+      color: isSelected ? Colors.blue : Colors.white,
+      elevation: isSelected ? _pickerSelectedElevation : 0.0,
+      onPressed: onTapped,
+    ),
+  );
+}
+
+Widget shapePicker({
+  double pickerHeight = _pickerRowHeight,
+  String selectedValue,
+  List<String> shapeNames,
+  IndexedValueCallback<String> onItemTapped,
+}) {
+  final List<String> shapeNameOptions = shapeNames ??
```

Comment:
Looks like shapeNames should be:
`enum ShapeNames { square, beveled, rounded, stadium }`

And shapeNameOptions would be:
` final List<String> shapeNameOptions = shapeNames ?? ShapeNames.values`;